### PR TITLE
fix: use accessor to enable sorting for 24h change col

### PR DESCRIPTION
**Pull request by evgenyboxer**
Fixes this:
Screen Shot 2020-07-21 at 9 53 00 pm

### CODE REVIEW

File: src/pages/Synths/Home/SynthsSection/SynthsTable/SynthsTable.tsx [hunk 1]
```diff
@@ -102,21 +102,15 @@ export const SynthsTable: FC<SynthsTableProps> = memo(
 							{
 								Header: <>{t('synths.home.table.24h-change-col')}</>,
 								sortType: 'basic',
+								accessor: (originalRow: any) =>
```
Comment (evgenyboxer) [inline src/pages/Synths/Home/SynthsSection/SynthsTable/SynthsTable.tsx:105]: Since react-table isn't 100% typed correctly, I'm using `any` here for now.

Comment (dvd-schwrtz) [reply src/pages/Synths/Home/SynthsSection/SynthsTable/SynthsTable.tsx:105]: why is it not typed correctly?

Comment (dvd-schwrtz) [reply src/pages/Synths/Home/SynthsSection/SynthsTable/SynthsTable.tsx:105]: you could also destructure originalRow here maybe just do `{ historicalRates }`

Comment (evgenyboxer) [reply src/pages/Synths/Home/SynthsSection/SynthsTable/SynthsTable.tsx:105]: its on my todo list... react-table v7 typing is quite complex with all the different plugins, etc...

**Pull request merged**